low-level/requirements.txt

```diff
@@ -1,10 +1,11 @@
-jsonschema==3.1.1
+jsonschema==3.2.0
```

Comment:
jsonschema is common pkg again, refer LRR2 specifications sheet, got updated in afternoon today.
same for python-consul.
Move installation of these to sspl dev deploy

Reply:
Done